namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-find-and-refer-an-intervention-dev/07-certificate.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-find-and-refer-an-intervention-ui-dev.hmpps.service.justice.gov.uk
+  namespace: hmpps-find-and-refer-an-intervention-dev
+spec:
+  secretName: find-and-refer-an-intervention-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - hmpps-find-and-refer-an-intervention-ui-dev.hmpps.service.justice.gov.uk
```

Comment:
I believe there is a character limit of 64 characters on dns names. Hence I believe we can shorten it to
find-refer-intervention-ui-dev.hmpps.service.justice.gov.uk

[custom-domain-certificate-character-limit](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/custom-domain-cert.html#certificate-character-limit)